### PR TITLE
feat: remove redundant travis builds

### PR DESCRIPTION
**Pull request by sendilkumarn**
This cleans up some travis build, since we have the GitHub Actions 

Part of #821

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -17,62 +17,21 @@ jobs:
...
-      env:
-        - DESC="findbugs"
-        - CMD="./mvnw install findbugs:check --projects '!scala-unit-tests,!jmh-scala-tests,!jmh-tests,!jcstress-tests,!p2-repository' --activate-profiles all -DskipTests=true --no-transfer-progress"
-
-    - jdk: oraclejdk8
-      env:
-        - DESC="checkstyle"
-        - CMD="./mvnw install checkstyle:check --activate-profiles all -DskipTests=true --no-transfer-progress"
-
-    - jdk: oraclejdk8
```
Comment (donraab) [inline .travis.yml:30]: Please leave the unit tests for Oracle JDK 8

File: .travis.yml [hunk 2]
```diff
@@ -17,62 +17,21 @@ jobs:
...
-      env:
-        - DESC="unit tests"
-        - CMD="./mvnw install"
-
     - jdk: oraclejdk8
```
Comment (donraab) [inline .travis.yml:20]: Remove the performance tests for Oracle JDK 8

File: .travis.yml [hunk 3]
```diff
@@ -17,62 +17,21 @@ jobs:
...
       env:
         - DESC="unit tests oraclejdk14"
         - CMD="./mvnw install --no-transfer-progress"
 
-    - jdk: openjdk-ea
-      env:
-        - DESC="unit tests openjdk-ea"
-        - CMD="./mvnw install --no-transfer-progress"
-
-    - jdk: oraclejdk8
```
Comment (donraab) [inline .travis.yml:70]: Please leave the unit tests for Java15-EA for now.

Comment (sendilkumarn) [reply .travis.yml:70]: This is the same as below :)

File: .travis.yml (hunk 1 above)
Comment (sendilkumarn) [reply .travis.yml:30]: Done

File: .travis.yml (hunk 2 above)
Comment (sendilkumarn) [reply .travis.yml:20]: Done

File: .travis.yml (hunk 3 above)
Comment (donraab) [reply .travis.yml:70]: Oh, I see. Thanks for clarifying.

Comment (sendilkumarn) [reply .travis.yml:70]: I mean we have the same test running in the `allow_failures` section.

**Comment by donraab** [comment]
@sendilkumarn Remove the OracleJDK 8 Performance Tests section as well and then I think this looks good.

**Comment by nikhilnanivadekar** [comment]
Please done remove any EA builds. They are repeated in 2 places in order to ensure failure is allowed.

File: .travis.yml (hunk 3 above)
Comment (nikhilnanivadekar) [reply .travis.yml:70]: That’s now how Travis works. Travis needs the actual build and also a section to allow failures. The allow failures has to be a copy paste for Travis to “allow failure”. Please revert this

Comment (sendilkumarn) [reply .travis.yml:70]: Oops is it? That looks strange.  I didn't know that. Okay will revert that for now.

Comment (nikhilnanivadekar) [reply .travis.yml:70]: Yeah took me about 20 iterations to figure out what I was missing when I set them up initially!

**Comment by nikhilnanivadekar** [comment]
This looks good to me once you remove the unnecessary whitespace. It being a yaml I just don’t want to risk it. Please rebase on to master. Lastly can you please update the commit message? It shows an entire log of changes related to this PR. Commit messages help while writing Release Notes. Thanks for your contribution!

**Comment by donraab** [comment]
Thanks for rebasing. I will merge as soon as the builds complete. @motlin @nikhilnanivadekar

**Comment by nikhilnanivadekar** [comment]
You needn’t wait for the build to finish. We know it works. Travis is backed up and it’ll be another few hours before it passes

**Pull request merged**